script.js

```diff
@@ -15,4 +15,137 @@ window.onload = () => {
   menu.addEventListener('click', () => {
     popUp.classList.add('hidden');
   });
-};
\ No newline at end of file
+
+  const projectsData = [
+    {
+      title: 'Tonic',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/second-section/project-01.svg',
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
```

Comment:
- [ ] please kindly add the popup image size for the desktop version so it looks like that of figma
- [ ] increase the close icon a little and bring it inside to make it more visible
- [ ] Kindly add your description font size(16px) like that of figma

yours
![Screenshot (435)_LI](https://user-images.githubusercontent.com/65298048/188755742-2ad9db31-a527-4762-abf2-87a9ba2e811d.jpg)
figma
![Screenshot (431)](https://user-images.githubusercontent.com/65298048/188755836-e5217d7e-53b8-44e4-8d04-30b5754e73fa.png)